File: ide/1.2.0/Phalconplus/Contracts/Auth/Access/DispatchEvent.zep.php
<?php

namespace PhalconPlus\Contracts\Auth\Access;

use Phalcon\Mvc\DispatcherInterface;
use Phalcon\Events\Event;

interface DispatchEvent
{

    /**
     * @param \Phalcon\Events\Event $event
     * @param \Phalcon\Mvc\DispatcherInterface $dispatcher
     * @return bool
     */
    public function beforeExecuteRoute(\Phalcon\Events\Event $event, \Phalcon\Mvc\DispatcherInterface $dispatcher): bool;

}

File: ext/phalconplus/contracts/auth/access/dispatchevent.zep.c

#ifdef HAVE_CONFIG_H
#include "../../../../ext_config.h"
#endif

#include <php.h>
#include "../../../../php_ext.h"
#include "../../../../ext.h"

#include <Zend/zend_exceptions.h>

#include "kernel/main.h"


ZEPHIR_INIT_CLASS(PhalconPlus_Contracts_Auth_Access_DispatchEvent)
{
	ZEPHIR_REGISTER_INTERFACE(PhalconPlus\\Contracts\\Auth\\Access, DispatchEvent, phalconplus, contracts_auth_access_dispatchevent, phalconplus_contracts_auth_access_dispatchevent_method_entry);

	return SUCCESS;
}

ZEPHIR_DOC_METHOD(PhalconPlus_Contracts_Auth_Access_DispatchEvent, beforeExecuteRoute);
